src/server/api/cla.js

```diff
@@ -524,9 +523,7 @@ async function claNotRequired(args, updateMethod) {
 // token (mandatory)
 async function updateUsersPullRequests(args) {
     try {
-        const user = await User.findOne({
-            name: args.user
-        })
+        const user = await userService.byUserName(args.name)
```

Comment:
@codezork we used `args.user` and you changed it to `args. name` ... is it on purpose

Reply:
You are right. I messed it up. Corrected now